### PR TITLE
Add sitemap generator

### PR DESCRIPTION
**Pull request by beagleknight**
# What and why

We need a `sitemap.xml` file for SEO purposes. I included a simple Rack middleware to generate a dynamic one.
# QA

Visit `/sitemap.xml` and check if it works properly.
# GIF Tax

### CODE REVIEW

**Comment by josepjaume** [comment]
Awesome!

File: lib/sitemap_middleware.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  end
+
+  private
+
+  def generate_sitemap
+    SitemapGenerator::Sitemap.default_host = "#{!ENV["FORCE_SSL"].blank? ? "https" : "http"}://#{ENV["SERVER_NAME"]}"
+
+    SitemapGenerator::Sitemap.create(adapter: self) do
+      # Add meetings pages
+      Meeting.all.each { |meeting| add meeting_path(meeting), lastmod: meeting.updated_at }
```
Comment (josepjaume) [inline lib/sitemap_middleware.rb]: `find_each` for better performance.

**Comment by beagleknight** [comment]
:+1:

**Comment by beagleknight** [comment]
@josepjaume Merge this pls!!!

**Comment by josepjaume** [comment]
:+1:

**Pull request merged**